### DISCUSSION TITLE
What the mean of setting Opacity to 2 (greater than 1)?

### DISCUSSION THREAD

**Discussion by GF-Huang** [HandyOrg/HandyControl#796] [Q&A] [answered]
What's the point of being greater than 1?

https://github.com/HandyOrg/HandyControl/blob/0762db0fb4b4a7378d8b8b783d09d30e7c306092/src/Shared/HandyControl_Shared/Themes/Styles/Base/ExpanderBaseStyle.xaml#L144-L146

**Comment by NaBian** [comment, marked as answer]
噢~这是明显的笔误

**Reply by GF-Huang** [reply]
我还以为会有什么神奇的用法。
